src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/custom.py

```diff
@@ -282,6 +284,17 @@ def new_cluster(client,
 
     return output_dict
 
+def _build_detailed_error(topError, outputList):
```

Comment:
Variables should be underscore cased in Python

Reply:
Thanks, still fairly new to python so other language styles still creep in